datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/base.py

```diff
@@ -131,6 +121,13 @@ def create_connection(self):
         self.log.debug('Setting `server` to `%s`', self._connection.server)
         self._connection.connect()
 
+    def setup_refresher(self):
+        self.OBJECT_REFRESHER.add_server(self._connection.server)
+
+        # Expected for multiple calls
+        with suppress(RuntimeError):
```

Comment:
Is this safe?

Reply:
From my understanding this is just to make sure the `OBJECT_REFRESHER` is started once, and anytime it's called after (ie other integrations) it doesn't matter since it's already started